receiver/k8sclusterreceiver/README.md

```diff
@@ -67,6 +67,9 @@ The following allocatable resource types are available.
   - storage
 - `metrics`: Allows to enable/disable metrics.
 - `resource_attributes`: Allows to enable/disable resource attributes.
+- `ignore_deprecated_resource` (default = `false`): Whether to ignore deprecated Kubernetes API resources. If set to true,
```

Comment:
`ignore_deprecated_resources` would probably be a better name? (I know I mentioned `ignore_deprecated_resource` in the issue :))

Reply:
agreed. That looks like better name.

Updated.